samples/drivers/counter/alarm/src/main.c

```diff
@@ -31,11 +31,12 @@ struct counter_alarm_cfg alarm_cfg;
 #define TIMER DT_LABEL(DT_INST(0, xlnx_xps_timer_1_00_a))
 #elif defined(CONFIG_COUNTER_ESP32)
 #define TIMER DT_LABEL(DT_NODELABEL(timer0))
+#elif defined(CONFIG_COUNTER_MCUX_CTIMER)
+#define TIMER DT_LABEL(DT_NODELABEL(ctimer0))
 #endif
 
-static void test_counter_interrupt_fn(const struct device *counter_dev,
-				      uint8_t chan_id, uint32_t ticks,
-				      void *user_data)
+static void test_counter_interrupt_fn(const struct device *counter_dev, uint8_t chan_id,
+				      uint32_t ticks, void *user_data)
```

Comment:
Please move all the whitespace changes to a separate commit

Reply:
Should be removed now.